browser.py

```diff
@@ -1,12 +1,14 @@
+#!/usr/bin/env python
+
 import json
 import logging
 
-import gobject
-import webkit
+from gi.repository import WebKit
+from gi import _gobject as gobject
```

Comment:
@jcollado I got no Idea, ;) I'm still learing Gtk3.
I was looking closest possible match for `gobject` module from `gi` and never even looked into `gi.repository` coz `_gobject` was not giving any trouble.